### PR TITLE
gstreamer: several updates

### PR DESCRIPTION
**Pull request by bilboed**
Should help detect leaks better and adds a new simple(r) target

### CODE REVIEW

File: projects/gstreamer/build.sh [hunk 1]
```diff
@@ -21,12 +21,20 @@ PREFIX=$WORK/prefix
 PLUGIN_DIR=$PREFIX/lib/gstreamer-1.0
 export PKG_CONFIG_PATH=$PREFIX/lib/pkgconfig
 mkdir -p $PREFIX
+export PATH=$PREFIX/bin:$PATH
 cd $WORK
 
 # Minimize gst-debug level/code
 export CFLAGS="$CFLAGS -DGST_LEVEL_MAX=2"
+#export LD=$CXX
```
Comment (Dor1s) [inline projects/gstreamer/build.sh]: I tried to remove this line via GitHub's web interface, and CLA started to complain, weird.

Could you please remove that and create a new PR, so it won't complain anymore?

Comment (bilboed) [reply projects/gstreamer/build.sh]: whoops, I'll remove it indeed

**Comment by bilboed** [comment]
googlebot, I did sign it !

File: projects/gstreamer/build.sh (hunk 1 above)
Comment (bilboed) [reply projects/gstreamer/build.sh]: Reuploaded with fixed commits. googlebot seems happy now :)

**Comment by Dor1s** [comment]
Great, thanks Edward!

**Pull request merged**